pkg/reconciler/cluster/controller.go

```diff
@@ -107,7 +107,9 @@ func NewController(cfg *rest.Config, syncerImage string, kubeconfig clientcmdapi
 			}
 			return []string{}, nil
 		},
-	})
+	}); err != nil {
+		klog.Fatalf("Failed to add indexer for APIResourceImport: %v", err)
```

Comment:
Do we really want to **exit** the calling executable here ? Isn't `Errorf` sufficient, especially if we think that this method should be available as a library to be included into other pieces of software ?
I'd rather add an error return value to the `NewController` function I assume.